client/src/components/BottomPanel.tsx

```diff
@@ -16,6 +16,11 @@ import SoundPanel from './SoundPanel';
 import { TowerDefensePanel } from './TowerDefensePanel';
 import { Weather } from './Weather';
 import WhiteboardPanel from './WhiteboardPanel';
+<<<<<<< HEAD
```

Comment:
more merge conflict code, if you're using an IDE like vscode it's easy to tell which files have conflicts.